app/models/world_location_news.rb

```diff
@@ -63,6 +63,10 @@ def translations
     end
   end
 
+  def ordered_translations
+    translations.sort_by { |t| t[:locale] == I18n.default_locale.to_s ? "" : t[:locale] }
```

Comment:
I'm not sure this is worth having as a separate method - think we could just in line this in the translations method, and maybe rename that to ordered translations? Think we have a bit of a tendency to do tiny methods but I don't think it makes it more readable.

So something like 

```
  def ordered_translations
    @ordered_translations ||= @content_item.content_item_data.dig("links", "available_translations")&.map do |translation|
      {
        locale: translation["locale"],
        base_path: translation["base_path"],
        text: I18n.t("shared.language_name", locale: translation["locale"]),
        active: I18n.locale.to_s == translation["locale"],
      }
    end.sort_by { |t| t[:locale] == I18n.default_locale.to_s ? "" : t[:locale] }
  end
```

If not, maybe we should make translations private to make it clear we should only be accessing the ordered version, but my preference would be to inline it